app/assets/stylesheets/course/courses.scss

```diff
@@ -51,6 +51,8 @@
         height: 100%;
         margin: -4px;
         padding: 4px;
+        position: absolute;
```

Comment:
The `a` tag extends to the space in between boxes. Would it be a good idea to add `position: relative` to `div.box.text-center.thumbnail`?

Reply:
Good catch - have fixed it.